packages/core/src/platform/platform.ts

```diff
@@ -135,16 +133,26 @@ export function assertPlatform(requiredToken: any): PlatformRef {
 /**
  * Returns the current platform.
```

Comment:
nit: rephrased a bit and suggested an alternative way of obtaining this info:
```suggestion
 * Returns the current platform in the browser environment. In the server environment,
 * returns `null`. If you need access to the platform info, inject `PlatformRef` in your application
 * code.
```

Reply:
If we use this content, we can remove the `@remarks` part.

Reply:
I removed it, although In general, I do prefer to use `@remarks` to callout important information.